docs/core-concepts/1220-permanent-cache.md

```diff
@@ -0,0 +1,154 @@
+---
+slug: /1220/permanent-cache-with-dagger
```

Comment:
The id will need updating, it currently clashes with https://docs.dagger.io/1220/vs